woocommerce/woocommerce
# Remove set up store details note

### Changes proposed in this Pull Request:

Closes #34341. Removes SetUpStoreDetails note and deletes the note instance in existing stores.

### How to test the changes in this Pull Request:

**Testing in an existing site**
1. Don't check out this branch yet
1. Reset onboarding with [woocommerce-admin-test-helper](https://github.com/woocommerce/woocommerce-admin-test-helper/releases/tag/v0.7.6) by going to `Tools > WCA Test Helper` and run `Tools > Reset Onboarding Wizard`
3. Go to Home > WooCommerce, it should show the OBW
4. Skip OBW
5. Observe the note `Add your store details to complete store setup` appears in Homescreen
6. Check out this branch
7. Go to `Tools > WCA Test Helper` and run `Trigger WCA Install`
8. Go to Home > WooCommerce, the note should be deleted

### Other information:



### FOR PR REVIEWER ONLY:
